cms/release_calendar/forms.py

```diff
@@ -66,13 +65,6 @@ def clean(self) -> dict:
             raise ValidationError({"next_release_date": "The next release date must be after the release date."})
 
         release_date_text = cleaned_data.get("release_date_text")
-        if cleaned_data.get("release_date") and release_date_text:
-            error = "Please enter the release date or the release date text, not both."
-            raise ValidationError({"release_date": error, "release_date_text": error})
-
-        if cleaned_data.get("next_release_date") and cleaned_data.get("next_release_text"):
-            error = "Please enter the next release date or the next release text, not both."
-            raise ValidationError({"next_release_date": error, "next_release_text": error})
```

Comment:
The next release date or next relese date text logic is still relevant, so this check should stay. Unless there was a change in requirements that I am not aware of?

Reply:
I went with the same mechanism as for the release date, where the date text overwrites the datetime for consistency. Arguably this case is different because `next_release_date` is nullable, but it might be counter intuitive to see it enforced for next date and not for date.